packages/astro/src/core/preview/vite-plugin-astro-preview.ts

```diff
@@ -24,7 +24,8 @@ export function vitePluginAstroPreview(settings: AstroSettings): Plugin {
 					return;
 				}
 
-				const pathname = stripBase(req.url!, base);
+				let strippedPathname = stripBase(req.url!, base);
```

Comment:
```suggestion
				const strippedPathname = stripBase(req.url!, base);
```
Can this be a const? Doesn't look like it's reassigned